components/actionbutton/index.css

```diff
@@ -0,0 +1,165 @@
+}
+
+.spectrum-ActionButton--L {
+  --spectrum-actionbutton-border-radius: var(--spectrum-actionbutton-l-border-radius);
+  --spectrum-actionbutton-border-size: var(--spectrum-actionbutton-l-border-size);
+  --spectrum-actionbutton-height: var(--spectrum-actionbutton-l-height);
+  --spectrum-actionbutton-hold-icon-padding-bottom: var(--spectrum-actionbutton-l-hold-icon-padding-bottom);
+  --spectrum-actionbutton-hold-icon-padding-right: var(--spectrum-actionbutton-l-hold-icon-padding-right);
+  --spectrum-actionbutton-icon-gap: var(--spectrum-actionbutton-l-icon-gap);
+  --spectrum-actionbutton-icon-size: var(--spectrum-actionbutton-l-icon-size);
+  --spectrum-actionbutton-min-width: var(--spectrum-actionbutton-l-min-width);
+  --spectrum-actionbutton-quiet-border-radius: var(--spectrum-actionbutton-l-quiet-border-radius);
+  --spectrum-actionbutton-quiet-border-size: var(--spectrum-actionbutton-l-quiet-border-size);
+  --spectrum-actionbutton-quiet-text-font-weight: var(--spectrum-actionbutton-l-quiet-text-font-weight);
+  --spectrum-actionbutton-quiet-text-size: var(--spectrum-actionbutton-l-quiet-text-size);
+  --spectrum-actionbutton-text-font-weight: var(--spectrum-actionbutton-l-text-font-weight);
+  --spectrum-actionbutton-text-padding-x: var(--spectrum-actionbutton-l-text-padding-x);
+  --spectrum-actionbutton-text-size: var(--spectrum-actionbutton-l-text-size);
+}
+
+.spectrum-ActionButton--XL {
+  --spectrum-actionbutton-border-radius: var(--spectrum-actionbutton-xl-border-radius);
+  --spectrum-actionbutton-border-size: var(--spectrum-actionbutton-xl-border-size);
+  --spectrum-actionbutton-height: var(--spectrum-actionbutton-xl-height);
+  --spectrum-actionbutton-hold-icon-padding-bottom: var(--spectrum-actionbutton-xl-hold-icon-padding-bottom);
+  --spectrum-actionbutton-hold-icon-padding-right: var(--spectrum-actionbutton-xl-hold-icon-padding-right);
+  --spectrum-actionbutton-icon-gap: var(--spectrum-actionbutton-xl-icon-gap);
+  --spectrum-actionbutton-icon-size: var(--spectrum-actionbutton-xl-icon-size);
+  --spectrum-actionbutton-min-width: var(--spectrum-actionbutton-xl-min-width);
+  --spectrum-actionbutton-quiet-border-radius: var(--spectrum-actionbutton-xl-quiet-border-radius);
+  --spectrum-actionbutton-quiet-border-size: var(--spectrum-actionbutton-xl-quiet-border-size);
+  --spectrum-actionbutton-quiet-text-font-weight: var(--spectrum-actionbutton-xl-quiet-text-font-weight);
+  --spectrum-actionbutton-quiet-text-size: var(--spectrum-actionbutton-xl-quiet-text-size);
+  --spectrum-actionbutton-text-font-weight: var(--spectrum-actionbutton-xl-text-font-weight);
+  --spectrum-actionbutton-text-padding-x: var(--spectrum-actionbutton-xl-text-padding-x);
+  --spectrum-actionbutton-text-size: var(--spectrum-actionbutton-xl-text-size);
+}
+
+a.spectrum-ActionButton {
+  @inherit: %spectrum-AnchorButton;
+}
+
+.spectrum-ActionButton {
+  @inherit: %spectrum-BaseButton;
```

Comment:
Now that we've broken this code out into BaseButton, we will have duplicate code, i.e. it will be defined for `.spectrum-ActionButton` as well as `.spectrum-Button`. This will increase output code size when both `@spectrum-css/button` and `@spectrum-css/actionbutton` are included in the same project.

Reply:
Not ideal, but gzipped shouldn't be much. Are you thinking we run a deduper?

Reply:
@GarthDB, nah, can't really de-dupe this. I don't think it's a big problem, RSP was asking for this anyway.